src/Feldspar/Onnx/onnxToFeld.hs

```diff
@@ -169,8 +169,23 @@ int2elemT :: Integral a => a -> TD.DataType
 int2elemT i = toEnum $ fromIntegral i :: TD.DataType
 
 showElemT :: TD.DataType -> String
-showElemT TD.FLOAT = "Float"
-showElemT t = error $ "Type " ++ show t ++ " not implemented"
+showElemT TD.FLOAT      = "Float"
+showElemT TD.UINT8      = "Word8"
+showElemT TD.INT8       = "Int8"
+showElemT TD.UINT16     = "Word16"
+showElemT TD.INT16      = "Int16"
+showElemT TD.INT32      = "Int32"
+showElemT TD.INT64      = "Int64"
+showElemT TD.STRING     = "String"
+showElemT TD.BOOL       = "Bool"
+showElemT TD.FLOAT16    = error $ "onnxToFeld.showElemT: FLOAT16 not implemented"
+showElemT TD.DOUBLE     = "Double"
```

Comment:
Is there nothing corresponding to our Float type in Feldspar?

Reply:
Not to FLOAT16. Of course, we can emulate it using a Float, but I did not lite to do that silently. We can of course do it later if we need to.

Reply:
Float was the first line in the function, I missed it since I was expecting to find it around the other floating point types.